src/doc_builder/meilisearch_helper.py

```diff
@@ -0,0 +1,138 @@
+import hashlib
+import sys
+from functools import wraps
+from time import sleep
+from typing import Callable, Tuple
+
+from meilisearch.client import Client, TaskInfo
+
+
+# References:
+# https://www.meilisearch.com/docs/learn/experimental/vector_search
+# https://github.com/meilisearch/meilisearch-python/blob/d5a0babe50b4ce5789892845db98b30d4db72203/tests/index/test_index_search_meilisearch.py#L491-L493
+# https://github.com/meilisearch/meilisearch-python/blob/d5a0babe50b4ce5789892845db98b30d4db72203/tests/conftest.py#L132-L146
+
+VECOR_NAME = "docs-embed"
+VECOR_DIM = 768
```

Comment:
```suggestion
VECTOR_NAME = "docs-embed"
VECTOR_DIM = 768
```
(nit) + propagate changes

Reply:
handled in https://github.com/huggingface/doc-builder/pull/506/commits/7cc64e32cb103f64776e3686f4c3eedec86293c4